vendor/gobierto_engines/custom-fields-progress-plugin/lib/tasks/calculate_progress.rake

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+namespace :gobierto_admin do
+  desc "Calculate progress"
+  task calculate_progress: :environment do
```

Comment:
Could you add another namespace with the plugin name at least? `gobierto_admin:calculate_progress` is very generic

Reply:
The namespaced version will be: `rake custom_field_plugins:progress:calculate_progress`